pkg/api/metadata/payer_info.go

```diff
@@ -28,9 +28,9 @@ type PayerInfoFetcher struct {
 	queries *queries.Queries
 }
 
-func NewPayerInfoFetcher(db *sql.DB) *PayerInfoFetcher {
+func NewPayerInfoFetcher(db *db.Handler) *PayerInfoFetcher {
 	return &PayerInfoFetcher{
-		queries: queries.New(db),
+		queries: db.ReadQuery(),
 	}
 }
```

Comment:
This is almost as a demonstration of passing in the read DB to a component that only does reads. However the component might evolve and in the future require a write query - to me it seems safer to just use `*db.Handler` everywhere.